test/NuGet.Core.Tests/NuGet.Commands.Test/RestoreCommand/FallbackFolderRestoreTests.cs

```diff
@@ -450,7 +451,15 @@ public async Task FallbackFolderRestore_VerifyMissingFallbackFolderFailsAsync()
 
                 // Act & Assert
                 var command = new RestoreCommand(request);
-                await Assert.ThrowsAsync<FatalProtocolException>(async () => await command.ExecuteAsync());
+                var result = await command.ExecuteAsync();
```

Comment:
Use explicit type instead of `var.

Reply:
fixed that instance, plus the 4-5 others that were in the file. next iteration.